src/Akka.Streams.Kafka/Stages/Consumers/Actors/KafkaConsumerActor.cs

```diff
@@ -755,59 +818,87 @@ private void ProcessError(Exception error)
             }
         }
 
-        private void Commit(IImmutableSet<TopicPartitionOffset> commitMap, Action<object> sendReply)
+        private void Commit(IImmutableSet<TopicPartitionOffset> commitMap, IImmutableSet<IActorRef> replyTo)
         {
+            var watch = Stopwatch.StartNew();
             try
             {
                 _commitRefreshing.UpdateRefreshDeadlines(commitMap.Select(tp => tp.TopicPartition)
                     .ToImmutableHashSet());
-
-                var watch = Stopwatch.StartNew();
+                _commitsInProgress += 1;
```

Comment:
`_commitsInProgress` for the JVM made more sense since their commits use `Task`s - ours are all synchronous so this probably will never be any value other than `1` or `0`.